app/scenes/Document/components/Header.tsx

```diff
@@ -211,7 +217,7 @@ function DocumentHeader({
           </>
         }
         actions={
-          <>
+          <Wrapper $hideHeader={hideHeader}>
             <ObservingBanner />
```

Comment:
Lets keep `<ObservingBanner />` visible

Reply:
Cool. Btw the navigation is always visible in collab editing mode. Do we want to fade out in it too?

Reply:
Yep, same behavior in collab is ideal